app/models/job.rb

```diff
@@ -20,4 +22,10 @@ class Job < ActiveRecord::Base
             presence: true,
             inclusion: { in: STATES }
   validates :author, presence: true
+
+  private
+
+  def set_published_at
+    self.published_at = DateTime.current
```

Comment:
I'll find how `.touch` save the date in the Database.

However, since touch is call in the background when a record is saved, we should be good, since it will be in the same timezone.